app/models.py

```diff
@@ -488,6 +488,10 @@ class Service(db.Model, Versioned):
     organisation = db.relationship('Organisation', backref='services')
 
     notes = db.Column(db.Text, nullable=True)
+    purchase_order_number = db.Column(db.String(255), nullable=True)
+    billing_contact_name = db.Column(db.Text, nullable=True)
+    billing_contact_email_address = db.Column(db.Text, nullable=True)
```

Comment:
I'm happy to change field names to `billing_contact_names` and `billing_contact_email_addresses`

The new table sounds interesting, the data would be more structured, but I also agree that it could be a slight overkill, and making layouts for displaying and editing it would be more faff.

Reply:
The only reason to use db.String(255) is to limit the length, there is no real difference between using db.text or  db.String(255) when it comes to performance or storage. Unless you have a reason to restrict the string length there is no advantage to using db.String(255). 255 seems an arbitrary number in this case.

Reply:
That makes sense @servingUpAces , good to know it does not make difference to performance.